sources/packages/backend/apps/api/src/services/education-program/education-program.service.ts

```diff
@@ -684,13 +702,19 @@ export class EducationProgramService extends RecordDataModelService<EducationPro
         deliveredOnSite: true,
         deliveredOnline: true,
       },
-      where: {
-        sabcCode: In(sabcCodes),
-        institution: {
-          id: institutionId,
+      where: [
```

Comment:
Do we still need the "[" since the `Or` was introduced?

Reply:
Yes, we don't  need it anymore. Thanks.

Reply:
We don't need it. Removed. Thanks.